.github/workflows/test-release.yaml

```
@@ -0,0 +1,45 @@
+name: Create test release
+on:
+  workflow_dispatch:
+    inputs:
+       version:
+        description: "create a test version for primazctl use"
+        required: true
+        default: "latest"
+
+jobs:
+  create_release:
```

I think it might make sense to run lints & tests during this as well; I'd prefer not to make a release that's known to have failing tests or lint checks.  This can probably be done in a future PR, but it's up to you.

---

This is just a test release, not a real one.  What it does is create a single file containing the primaza manifests for testing and as an interim until the primaza repository starts doing it. 
Example here:  https://github.com/mmulholla/primazactl/releases/tag/1.0.0

---

In that case, I'd suggest naming this job to be more indicative of that - I see the job title is "Create test release", but I'm hoping to avoid as much confusion as possible.  Something like this?
```suggestion
  create_fake_release:
```

mmulholl: git will not let me respond in a separate comment. Did this and added fake in more places.

---

Done.